pkg/minikube/node/config.go

```diff
@@ -127,12 +129,24 @@ func setupKubeAdm(mAPI libmachine.API, cfg config.ClusterConfig, node config.Nod
 }
 
 func setupKubeconfig(h *host.Host, c *config.ClusterConfig, n *config.Node, clusterName string) (*kubeconfig.Settings, error) {
-	addr, err := h.Driver.GetURL()
-	if err != nil {
-		exit.WithError("Failed to get driver URL", err)
-	}
-	if !driver.IsKIC(h.DriverName) {
+	addr := ""
+	var err error
+	if driver.IsKIC(h.DriverName) {
+		p, err := oci.HostPortBinding(h.DriverName, h.Name, n.Port)
```

Comment:
For another PR perhaps, but `HostPortBinding` may be a poor name for this function, as it sounds like it may be binding to a port rather than fetching information. Perhaps:

`oci.ForwardedPort` or `oci.BoundPort`, `oci.HostPort`?

Reply:
created an issue for it